python/kfserving/test/test_cred_utils.py

```diff
@@ -0,0 +1,19 @@
+from unittest import mock
+
+from kubernetes.client import V1ServiceAccountList, V1ServiceAccount, V1ObjectMeta
+
+from kfserving.api.creds_utils import check_sa_exists
+
+
+@mock.patch('kubernetes.client.CoreV1Api.list_namespaced_service_account')
+def test_check_sa_exists(mock):
```

Comment:
lint error:
INFO|2020-03-07T02:24:22|util.py:72| 
W:  9,25: Redefining name 'mock' from outer scope (line 1) (redefined-outer-name)

Reply:
Changed name to resolve lint error